modules/xmpp/SdpConsistency.js

```diff
@@ -78,6 +38,9 @@ export default class SdpConsistency {
      *  in future calls to makeVideoPrimarySsrcsConsistent
      */
     setPrimarySsrc (primarySsrc) {
+        if (typeof primarySsrc !== 'number') {
```

Comment:
In other comment you said that the code should fail loudly and that was my intention here. I added this Error after seeing that number and string types are mixed up. It helped me to fix the problem, before committing any broken code.